### ISSUE TITLE
Not accounting for switch statements

### ISSUE THREAD

**Issue by CamBurris** [cflint/CFLint#234]
It seems like the control expression in a switch statement is not being accounted for.

For example, the following code will trigger a false positive UNUSED_METHOD_ARGUMENT error. 
```
<cfscript>
	public struct function onLoad(boolean test = false) output="false" {
		switch (arguments.test) {
		}

		return {};
	}
</cfscript>
```

**Comment by CamBurris** [comment]
It seems like we are also not taking into account the body of the switch statement. 

The below will also trigger an UNUSED_METHOD_ARGUMENT.
```
<cfscript>
	public struct function onLoad(boolean test = false) output="false" {
		switch (true) {
			case true:
				if (arguments.test) {
				}
		}
	}
</cfscript>
```

**Comment by ryaneberly** [comment]
Good catch.  It is fixed.
